pyuvdata/uvbeam/uvbeam.py

```diff
@@ -212,11 +218,13 @@ def __init__(self):
         )
 
         desc = (
-            "Beam basis vector components -- directions for which the "
-            "electric field values are recorded in the pixel coordinate system. "
+            "Beam basis vector components, essentially the mapping between the "
+            "directions that the electrical field values are recorded in to the "
+            "directions aligned with the pixel coordinate system (or azimuth/zenith "
+            "angle for HEALPix beams)."
             'Not required if beam_type is "power". The shape depends on the '
             'pixel_coordinate_system, if it is "healpix", the shape is: '
-            "(Naxes_vec, Ncomponents_vec, Npixels), otherwise it is "
+            "(Naxes_vec, Ncomponents_vec=2, Npixels), otherwise it is "
             "(Naxes_vec, Ncomponents_vec, Naxes2, Naxes1)"
         )
         self._basis_vector_array = uvp.UVParameter(
```

Comment:
Why is acceptable range (0,1)? Shouldn't it be at least (-1,1)?

Reply:
You are correct, it should be (-1,1). Thanks for catching that!